pkg/storageincentives/agent.go

```diff
@@ -277,6 +277,8 @@ func (a *Agent) handleCommit(ctx context.Context, round uint64) (bool, error) {
 		return false, err
 	}
 
+	a.state.SetLastPlayedRound(round)
+
```

Comment:
Nice, we've played once we've successfully committed.

Reply:
+1, this is the best place to set last played round